### DISCUSSION TITLE
Javascript Error in dropdown

### DISCUSSION THREAD

**Discussion by Knud13** [twbs/bootstrap#40658] [Q&A] [answered]
I've found a small javascript error in the dropdown-menu e.g.
```
<div class="dropdown-menu">
  <a class="dropdown-item" href="#">New around here? Sign up</a>
  <a class="dropdown-item" href="#">Forgot password?</a>
</div>
```
Keyboard Tab into the dropdown menu works but if you then use the arrow keys it will generate an error

>  Uncaught TypeError: Cannot read properties of undefined (reading 'parentNode') dropdown.js:98
    at new qi (dropdown.js:98:34)
    at qi.getOrCreateInstance (base-component.js:65:41)
    at HTMLDivElement.dataApiKeydownHandler (dropdown.js:419:31)
    at HTMLDocument.n (event-handler.js:118:19)

you can reproduce the error on the bootstrap docs page 
[https://getbootstrap.com/docs/5.3/components/dropdowns/#forms](https://getbootstrap.com/docs/5.3/components/dropdowns/#forms)
just navigate to the "New around here? Sign up" dropdown-item via keyboard so it has focus and use the arrow keys.

I assume the dropdown-item is looking for a parent <li> since most dropdown-item are in the format
```
<ul class="dropdown-menu">
  <li><a class="dropdown-item" href="#">New around here? Sign up</a></li>
  <li><a class="dropdown-item" href="#">Forgot password?</a></li>
</ul>
```

**Comment by julien-deramond** [comment, marked as answer]
Hello there,
I've provided an answer in https://github.com/twbs/bootstrap/issues/40660#issuecomment-2246954122
